website/docs/docs/build/custom-schemas.md

```diff
@@ -62,7 +60,10 @@ Further, the schema that your development models are built in would be the same
 If you prefer to use different logic for generating a schema name, you can change the way dbt generates a schema name (see below).
 
 ### How does dbt generate a model's schema name?
-Under the hood, dbt uses a macro called `generate_schema_name` to determine the name of the schema that a model should be built in. The code for the macro that expresses the current logic follows:
+
+Under the hood, dbt uses a default macro called `generate_schema_name` to determine the name of the schema that a model should be built in. 
```

Comment:
Trying to weed out idioms that might cause problems for non-native speakers and (eventually) translations as we go. 

```suggestion
dbt uses a default macro called `generate_schema_name` to determine the name of the schema that a model should be built in. 
```